mkdocs.yml

```diff
@@ -44,7 +44,9 @@ nav: # Navigation structure for the documentation
     - Intrinsic periodicity of the SIR system: notebooks/06_SIR_wbirths_natural_periodicity.ipynb
     - Exploring the critical community size of an SIR model: notebooks/07_SIR_CCS.ipynb
     - The relationship between spatial coupling and incidence correlation in a 2-patch model: notebooks/08_2patch_SIR_wbirths_correlation.ipynb
-  - API Reference: api.md # include all the autogenerated markdown files
+  - API Reference:
```

Comment:
Can you made this "API reference" to stick with sentence casing for headings and titles?